ui/apps/platform/src/Containers/Integrations/utils/tableColumnDescriptor.ts

```diff
@@ -201,6 +201,15 @@ const tableColumnDescriptor: Readonly<IntegrationTableColumnDescriptorMap> = {
                     integration.syslog.tcpConfig.skipTlsVerify ? 'Yes (Insecure)' : 'No (Secure)',
             },
         ],
+        microsoftSentinel: [
+            { accessor: 'name', Header: 'Name' },
+            {
+                accessor: 'microsoftSentinel.logIngestionEndpoint',
+                Header: 'Log Ingestion Endpoint',
+            },
+            { accessor: 'microsoftSentinel.directoryTenantId', Header: 'Directory Tenant ID' },
+            { accessor: 'microsoftSentinel.applicationClientId', Header: 'Application Client ID' },
+        ],
```

Comment:
Nit: table headers should be sentence case to match the style recommendations and other sections of the app.
```suggestion
        microsoftSentinel: [
            { accessor: 'name', Header: 'Name' },
            {
                accessor: 'microsoftSentinel.logIngestionEndpoint',
                Header: 'Log ingestion endpoint',
            },
            { accessor: 'microsoftSentinel.directoryTenantId', Header: 'Directory tenant ID' },
            { accessor: 'microsoftSentinel.applicationClientId', Header: 'Application client ID' },
        ],
```